SweetsFactory/src/app/shared/navbar/navbar.component.html

```diff
@@ -1,8 +1,8 @@
-<div class="grid-y navbar-grid">
+<div class="grid-y navbar-grid ">
     <div class="cell shrink">
-        <h3 class="menu-title">{{menuTitle | lowercase }}</h3>
+        <h3 class="menu-title text-center">{{menuTitle | lowercase }}</h3>
```

Comment:
Why did you use a _lowercase_ filter here? Wouldn't a _Capitalize_ filter be more suitable? :)